pywinauto/controls/uiawrapper.py

```diff
@@ -131,7 +131,7 @@ def __get__(self, obj, cls):
         if obj is None:
             return None
         value = self.fget(obj)
-        setattr(obj, self.func_name, value)
+        #setattr(obj, self.func_name, value)
```

Comment:
What is the reason to comment this line?

Reply:
It was internal debug accidentally committed.